test/variable.jl

```diff
@@ -70,11 +70,13 @@ facts("[variable] repeated elements in index set (issue #199)") do
 end
 
 # Test conditionals in variable definition
-# condmod = Model()
-# @defVar(condmod, x[i=1:10]; iseven(i))
-# @defVar(condmod, y[j=1:10,k=3:2:9]; isodd(j+k) && k <= 8)
-# @test JuMP.dictstring(x, :REPL)   == "x[i], for all i in {1..10} s.t. iseven(i) free"
-# @test JuMP.dictstring(x, :IJulia) == "x_{i} \\quad \\forall i \\in \\{ 1..10 \\} s.t. iseven(i) free"
-# @test JuMP.dictstring(y, :REPL)   == "y[i,j], for all i in {1..10}, j in {3,5..7,9} s.t. isodd(j + k) and k <= 8 free"
-# @test JuMP.dictstring(y, :IJulia) == "y_{i,j} \\quad \\forall i \\in \\{ 1..10 \\}, j \\in \\{ 3,5..7,9 \\} s.t. isodd(j + k) and k <= 8 free"
-# @test string(condmod) == "Min 0\nSubject to \nx[i], for all i in {1..10} s.t. iseven(i) free\ny[i,j], for all i in {1..10}, j in {3,5..7,9} s.t. isodd(j + k) and k <= 8 free\n"
+condmod = Model()
```

Comment:
We should run these tests only on 0.4, and also add a test that catches any changes in parsing.